python/ray/worker.py

```diff
@@ -177,8 +170,8 @@ def __init__(self):
         # Reads/writes to the following fields must be protected by
         # self.state_lock.
         # Identity of the driver that this worker is processing.
-        self.task_driver_id = ray.ObjectID(NIL_ID)
-        self.current_task_id = ray.ObjectID(NIL_ID)
+        self.task_driver_id = ray.ObjectID()
```

Comment:
instead of using `ray.ObjectID` everywhere, it'd be nicer to `from ray import ObjectID` and just use `ObjectID`